server/command.go

```diff
@@ -517,9 +474,8 @@ func executeUninstallServer(p *Plugin, c *plugin.Context, header *model.CommandA
 		&model.WebsocketBroadcast{},
 	)
 
-	const uninstallInstructions = `Jira instance successfully disconnected. Go to **Settings > Applications > Application Links** to remove the application in your Jira Server or Data Center instance.`
-
-	return p.responsef(header, uninstallInstructions)
+	uninstallInstructions := `Jira instance successfully disconnected. Navigate to [**your app management URL**](%s) in order to remove the application from your Jira instance.`
```

Comment:
nit: maybe we should change `disconnected` to `uninstalled` to be consistent. @aaronrothschild ?